tests/neg/i1665-nonfuzzy.scala

```diff
@@ -0,0 +1,5 @@
+
+object Test {
+  !=(1) // error: Binary operation != should have an operand on the left hand side
```

Comment:
In scalac there is a warning and then it is converted to `true`

```
<console>:12: warning: comparing values of types type and Int using `!=' will always yield true
       !=(1)
         ^
res0: Boolean = true
```
I could:
* Make it a warning and collapse it to `true` if in Scala 2 mode.
* Or make it a warning and collapse it to `true` in all cases.